src/content/blog/ja/zundamon-lt-project.md

```
@@ -0,0 +1,138 @@
+description: "技術の創造的活用！Marp、VOICEVOX、VTubeStudioを組み合わせて、ずんだもんによる自動LT発表システムを構築した実験プロジェクト"
+pubDate: 2024-01-30
+author: "Terisuke"
+category: "lab"
+tags: ["Marp", "VOICEVOX", "VTubeStudio", "自動化", "創造的プロジェクト"]
+# image:
+#   url: "/images/blog/zundamon-project.avif"
+#   alt: "ずんだもんLTプロジェクト"
+lang: "ja"
+---
+
+# 【Marp×VOICEVOX×VTubeStudio】ずんだもんにLT発表してもらった話
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)